toolkit/resources/manifests/package/toolchain_x86_64.txt

```diff
@@ -148,6 +148,9 @@ libassuan-devel-2.5.5-2.cm2.x86_64.rpm
 libcap-2.60-1.cm2.x86_64.rpm
 libcap-debuginfo-2.60-1.cm2.x86_64.rpm
 libcap-devel-2.60-1.cm2.x86_64.rpm
+libcap-ng-0.8.2-1.cm2.x86_64.rpm
```

Comment:
I'm on the fence about this, but I think we should bump the release number of `libcap-ng`. I think it makes sense to do a symbolic bump, since we're changing the environment it's built in. This also ensures that the latest version published on PMC matches what's used to build every other package (now that `libcap-ng` is in the worker chroot).